### PR TITLE
Streamlined the setup process

### PR DESCRIPTION
**Pull request by bwagnerr**
These are changes to simplify the setup and remove excess setup files.

I made sure we still support setuptools and pip and I documented in the README a simple way of setting up the keymanager.

Please note that even if it installs a specific library from the requirements, develop mode would still have precedence over it, so we can add all the leap dependencies we need directly to the requirements files

I also changed requirements-latest to requirements-pixelated to make it's purpose more explicit.

### CODE REVIEW

File: pkg/requirements.pip [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # if we bump the gnupg version, bump also the sanity check
 # in keymanager.__init__
+
+https://launchpad.net/dirspec/stable-13-10/13.10/+download/dirspec-13.10.tar.gz
 gnupg>=1.4.0
 simplejson
 requests
+leap.common>=0.4.3
```
Comment (kalikaneko) [inline pkg/requirements.pip]: we don't want leap dependencies in the main requirements, they should go in requirements-leap. this causes problems when we bump dependencies during a release cycle and the required dependency has not yet been published.

**Comment by kalikaneko** [comment]
> along with any other leap libraries now, it supports both setuptools and pip as means of setting up dependencies

I see you're removing requirements-leap in this setup. what problem is this removal addressing?
requirements-leap was added to address a particular situation in our development workflow, and if it's not strictly needed I would prefer not to remove it.

**Comment by bwagnerr** [comment]
Kaliy, basically I've seen people confused with the setup process all the time. Most people with some python background expect one of two things, being able to pip install -r requirements or python setup.py develop and be ready to develop.
This is not true today, if you try to run the requirements or develop, everything breaks because you won't have all dependencies yet.

As I understand, separating the requirements_leap was made so you could install all libraries in develop mode (using the develop_mode thingie that you have), the thing is that you can leave the requirements on the main requirements.pip file and still install the library from the library folder in develop mode, and it would use the develop mode one, so this shouldn't disrupt your current workflow, while also allowing for the pip workflow, removing all complexity beside that.

**Comment by zaraG** [comment]
I just added a line in the README which fixes a bug in the gnupg library for mac users.

**Comment by kalikaneko** [comment]
something weird has happen with this PR.
Originally, it was a commit by @bwagnerr about streamlining the setup process.
I got a notification today about it being merged, but I only see commit 2d69975880214e in here.
Somebody has an explanation for this github mess?

**Comment by zaraG** [comment]
You are right this looks really strange, the PR which is merged should be this one: https://github.com/leapcode/keymanager/pull/112
I wonder what happend with the changes of @bwagnerr

**Pull request merged**